src/scss/components/button/mixins/_hxButton.scss

```diff
@@ -178,6 +178,7 @@
   background-color: transparent;
   border: none;
   color: $blue-500;
+  padding: 0.56rem 0 0.43rem;
```

Comment:
### 1. Please revert change to:

```css
padding: 0;
```

### 2. Change:

https://github.com/HelixDesignSystem/helix-ui/blob/83121046703bd6bc0c58f3c8da5d999227208d4b/src/scss/components/button/_buttonSet.scss#L5

to:

```css
margin-left: 1rem;
```

### 3. Add padding for Button Set Tertiary button here:

https://github.com/HelixDesignSystem/helix-ui/blob/83121046703bd6bc0c58f3c8da5d999227208d4b/src/scss/components/button/_buttonSet.scss#L1-L7

Reply:
Changes have been done, please review.